stdlib/tkinter/ttk.pyi

```diff
@@ -1023,29 +1021,35 @@ class Treeview(Widget, tkinter.XView, tkinter.YView):
         anchor: tkinter._Anchor = ...,
         # id is read-only
     ) -> _TreeviewColumnDict | None: ...
-    def delete(self, *items: str) -> None: ...
-    def detach(self, *items: str) -> None: ...
-    def exists(self, item: str) -> bool: ...
+    @overload
+    def delete(self, items: str | int) -> None: ...
```

Comment:
`treeview.delete(items=foo)` will not work, so the `items` argument should be marked as positional-only. Then its name does not matter, so we might as well make it obvious that you are specifying just one item in this overload.

```suggestion
    def delete(self, __item: str | int) -> None: ...
```